frontend/src/pages/RegisterPage.tsx

```diff
@@ -126,7 +127,7 @@ export const RegisterPage = () => {
     if (userActions.register.rejected.match(res)) {
       errorHandler(res.payload?.response?.data);
     } else if (userActions.register.fulfilled.match(res)) {
-      history.push('/login');
+      history.push(paths.loginPage + search);
```

Comment:
I think this would be the place to check the redirect, if that would be done.

Also, the path for history, or the `to` property for links, can be provided as an object:
```suggestion
      history.push({ pathname: paths.loginPage, search });
```